bot/code_review_bot/report/phabricator.py

```diff
@@ -443,11 +430,31 @@ def pluralize(word, nb):
 
         assert comment != "", "Empty comment"
 
+        known_stats = defaultdict(int)
+        for i in known_issues:
+            known_stats[i.level.value] += 1
+
+        # Display more information in the footer section
+        comment += "\n\n---\n\n"
+
+        # Publish the number of known issues in case running with before/after
+        if revision.before_after_feature and known_stats:
+            known_issues_msg = " and ".join(
+                self.pluralize(level, nb) for level, nb in known_stats.items()
+            )
+            plural = sum(known_stats.values()) > 1
+            known_issues_msg = (
+                known_issues_msg
+                + f" {'were' if plural else 'was'} detected but already existing on the mozilla-central repository.\n"
+            )
+            comment += known_issues_msg
```

Comment:
Not sure it's worth adding this additional information as it isn't actionable if we don't give details.
Perhaps we could add it in the future once we have a way in the frontend to show the full list of pre-existing issues on a given revision.
Wdyt @abpostelnicu?